### DISCUSSION TITLE
Which branch for pre-release testing?

### DISCUSSION THREAD

**Discussion by david-bakin** [vector-of-bool/dds#90] [Q&A] [answered]
If I want to experiment with `dds`'s new any-test-driver feature (e.g., trying out `doctest`) - which branch do I build?  (In fact, is there a CI location where I can get that branch's built-artifacts?)

**Comment by vector-of-bool** [comment, marked as answer]
Hello!

At time of writing, the `develop` branch is used to contain to-be-released features. We do not (yet) use a "pending release" branch.

Pre-built static-linked binaries for Windows, macOS, and Linux on any branch can be downloaded from the Azure Pipelines project: https://dev.azure.com/vector-of-bool/dds/_build?definitionId=5
